db/collection.go

```diff
@@ -42,20 +41,7 @@ type Collection struct {
 	indexes      map[string]Index
 }
 
-func newCollection(name string, defaultInstance interface{}, d *DB) *Collection {
-	schema := jsonschema.Reflect(defaultInstance)
-	schemaLoader := gojsonschema.NewGoLoader(schema)
-	c := &Collection{
-		name:         name,
-		schemaLoader: schemaLoader,
-		valueType:    reflect.TypeOf(defaultInstance),
-		db:           d,
-		indexes:      make(map[string]Index),
-	}
-	return c
-}
-
-func newCollectionFromSchema(name string, schema string, d *DB) *Collection {
+func newCollection(name string, schema string, d *DB) *Collection {
```

Comment:
This is where we'll want some schema verification of `ID`

https://github.com/textileio/go-threads/issues/281